homeassistant/components/schluter/climate.py

```diff
@@ -0,0 +1,171 @@
+"""Support for Schluter thermostats."""
+from datetime import timedelta
+import logging
+
+from requests import RequestException
+import voluptuous as vol
+
+from homeassistant.components.climate import PLATFORM_SCHEMA, ClimateDevice
+from homeassistant.components.climate.const import (
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import ATTR_TEMPERATURE, CONF_SCAN_INTERVAL, TEMP_CELSIUS
+
+from . import DATA_SCHLUTER, DOMAIN as SCHLUTER_DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(seconds=5)
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {vol.Optional(CONF_SCAN_INTERVAL): vol.All(vol.Coerce(int), vol.Range(min=1))}
+)
+
+MODE_SCHLUTER_TO_HASS = {
+    False: HVAC_MODE_OFF,
+    True: HVAC_MODE_HEAT,
+}
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Schluter thermostats."""
+    data = hass.data[DATA_SCHLUTER]
+    devices = []
+    temp_unit = hass.config.units.temperature_unit
+
+    for thermostat in data.thermostats:
+        devices.append(SchluterThermostat(thermostat, temp_unit, data))
+        _LOGGER.debug("Adding thermostat: %s", thermostat.name)
```

Comment:
This is already logged by the core (the creation of entities).

Reply:
I was not aware of this, thanks! Removed this unnecessary log statement.